### PR TITLE
Added new seadfarmer module for Question and Answering Gen AI construct

### PR DESCRIPTION
**Pull request by saikatak**
## Describe your changes
Added new seadfarmer module for Question and Answering Gen AI construct

## Checklist before requesting a review

- [x] I updated `CHANGELOG.MD` with a description of my changes
- [ ] If the change was to a module, I ran the code validation script (`scripts/validate.sh`)
- [ ] If the change was to a module, I have added thorough tests
- [ ] If the change was to a module, I have added/updated the module's README.md
- [x] If a module was added, I added a reference to the module to the repository's README.md
- [x] I verified that my code deploys successfully using `seedfarmer apply`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by malachi-constant** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: AIOpsModuleTestsInfrastruct-doMLXEYsnmxr
* Commit ID: 269a352fc1b1b4065184b3877b048a23d553f2bf
* Result: SUCCEEDED
* [Build Logs](https://ythfngxmh6.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=0dd05405-230e-4ef6-ba70-2b50e494e6b4%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by malachi-constant** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: AIOpsModuleTestsInfrastruct-doMLXEYsnmxr
* Commit ID: cbcca91d9c79b9f6c11ddc9247d800828afe3601
* Result: SUCCEEDED
* [Build Logs](https://ythfngxmh6.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=c400a9dc-6187-4351-8b28-2b2c5cfdeb0f%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: modules/fmops/qna-rag/stack.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        **kwargs,
+    ) -> None:
+        super().__init__(
+            scope,
+            id,
+            description=" This stack creates resources for the LLM - QA RAG ",
+            **kwargs,
+        )
+
+        print(os_domain_endpoint)
```
Comment (kukushking) [inline modules/fmops/qna-rag/stack.py:31]: nitpick - please remove print statement

**Comment by malachi-constant** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: AIOpsModuleTestsInfrastruct-doMLXEYsnmxr
* Commit ID: 4957fb4516d66c9ba2e7809805d116a2a9fb2945
* Result: SUCCEEDED
* [Build Logs](https://ythfngxmh6.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=e04ed690-3747-407a-b4f9-52502e19862e%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: modules/fmops/qna-rag/stack.py (hunk 1 above)
Comment (saikatak) [reply modules/fmops/qna-rag/stack.py]: removed the print

**Comment by malachi-constant** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: AIOpsModuleTestsInfrastruct-doMLXEYsnmxr
* Commit ID: c4a831f318014787fd83ba9198917dc0bc8a8eb7
* Result: SUCCEEDED
* [Build Logs](https://ythfngxmh6.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=d34d8dde-245f-40c9-b35b-094d17cf31fe%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Pull request closed without merging**